CSS/index.css

```diff
@@ -28,6 +28,10 @@
   align-items: center;
   margin-top: 10px;
   border: 2.5px solid #931d25;
+  transform: translateY(1);
+  transform: rotate(360deg); /* cant get this to work */
```

Comment:
The reason this isn't working is because you are using mixed transform properties. You are telling it to go from rotated 360degrees to translated up 300px. The element is only doing what you are telling it to. There are a few ways to fix this. Just keep playing with it.